cumulusci/tasks/command.py

```diff
@@ -68,28 +81,46 @@ def _handle_returncode(self, returncode, stderr):
             self.logger.error(message)
             raise CommandException(message)
 
-    def _run_command(self, env):
+    def _run_command(self, env, command=None, output_handler=None, return_code_handler=None):
```

Comment:
Check out `_poll_action` on SFDXJsonPollingTask.  It injects a different command into _run_command to do a status check on the deployment.  In this case you don't want to change the command in self.options b/c that's the main command you're running.